integrations/gen_docs_integrations.py

```diff
@@ -0,0 +1,276 @@
+import json
+import os
+
+# Dictionary responsible for making the symbolic links at the end of the script's run.
+symlink_dict = {}
+
+
+def generate_category_from_name(category_fragment, category_array):
+    """
+    Takes a category ID in splitted form ("." as delimiter) and the array of the categories, and returns the proper category name that Learn expects.
+    """
+
+    category_name = ""
+    i = 0
+    dummy_id = category_fragment[0]
+
+    while i < len(category_fragment):
+        for category in category_array:
+
+            if dummy_id == category['id']:
+                category_name = category_name + "/" + category["name"]
+                try:
+                    # print("equals")
+                    # print(fragment, category_fragment[i+1])
+                    dummy_id = dummy_id + "." + category_fragment[i+1]
+                    # print(dummy_id)
+                except:
```

Comment:
```suggestion
                except IndexError:
```

I’m not 100% certain that this is the correct exception to match on here, but from looking at the code it seems to be the only one that should be possible.